src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const divError = `
+  <div class="error-message">
+    Promise was rejected!
+  </div>
+`;
+const divSuccess = `
+  <div class="message">
+    Promise was resolved!
+  </div>
+  `;
```

Comment:
Those layout are similar, maybe you can create a function that will return layout with args(like `className` and `text`)?